client/__tests__/index.js

```diff
@@ -0,0 +1,7 @@
+// import './specHelper.js'
```

Comment:
Not in the boilerplate, but more than likely in any project that uses it.  I left this here to show both places where it would need to be added back in.